pom.xml

```diff
@@ -259,9 +259,37 @@
                           <environmentVariables>
                               <AWS_PROFILE>${AWS_PROFILE}</AWS_PROFILE>
                               <AWS_REGION>${AWS_REGION}</AWS_REGION>
-                              <S3_BUCKET>${S3_BUCKET}</S3_BUCKET>
-                              <S3_DIR>${S3_DIR}</S3_DIR>
                           </environmentVariables>
+                          <properties>
+                              <io.jenkins.plugins.artifact_manager_s3.S3BlobStore.container>${S3_BUCKET}</io.jenkins.plugins.artifact_manager_s3.S3BlobStore.container>
```

Comment:
Huh? Why are there two profiles here? Pick one or the other.

Reply:
backward compatibility for testing

Reply:
ok, there is no release so it does not make sense.